postgres/datadog_checks/postgres/connections.py

```diff
@@ -4,11 +4,25 @@
 import datetime
 import inspect
 import threading
-from collections import namedtuple
+from typing import Callable, Dict
 
 import psycopg2
 
-ConnectionWithTTL = namedtuple("ConnectionWithTTL", "connection deadline")
+
+class ConnectionInfo:
```

Comment:
There's some syntax in this PR which is python3-compatible only. You might want to check with Agent Integrations (#agent-integrations) that the migration off of 2.7 is far enough along that it's acceptable. We're close, I'm just not sure if we've officially ripped the band-aid off yet.